forc-plugins/forc-tx/src/lib.rs

```diff
@@ -62,10 +67,19 @@ pub struct Create {
 pub struct Mint {
     /// The location of the `Mint` transaction in the block.
     #[clap(long)]
-    pub tx_ptr: fuel_tx::TxPointer,
-    // Outputs must follow all other arguments and are parsed separately.
+    pub tx_ptr: fuel_tx::TxPointer, // TODO: @hal3e check if the `clap` settings are ok
```

Comment:
The `Mint` transaction changed. Can someone please have a look at this part of the code ?

Reply:
We don't need to support `Mint` transactions on the `forc` level. Because it is not allowed to send transactions like this. It is only produced by block producer

Reply:
I removed the `Mint` variant